planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -1094,6 +1096,24 @@ const std::string& moveit::planning_interface::MoveGroup::getName() const
   return impl_->getOptions().group_name_;
 }
 
+const std::vector<std::string> moveit::planning_interface::MoveGroup::getNamedTargets()
+{
+  std::vector<std::string> output;
+  const robot_model::RobotModelConstPtr& robot = getRobotModel();
+  std::string group = getName();
+  const robot_model::JointModelGroup* joint_group = robot->getJointModelGroup(group);
+
+  if (joint_group)
+  {
+    const std::vector<std::string>& known_states = joint_group->getDefaultStateNames();
+    if (!known_states.empty())
+    {
+      output.insert(output.end(), known_states.begin(), known_states.end());
```

Comment:
im not sure, but is it necessary to copy this vector into a temporary vector, then return that vector?